### DISCUSSION TITLE
overall Medley Interlisp objectives

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#354] [General]
I wanted to talk about the bibliography project, not just what we’ve been doing on Zotero but the “bigger picture”.
I’m thinking it’s not just “making a medley bibliography, but also
-	Contributing bibliography technology and ideas on how to build one and maintain it in a group like ours
-	Training and documentation to support contractors (fiverr) 
-	Coordinate gathering of metadata with “Content Authenticity Initiative” for authoritative references
-	Link a timeline of features (DWIM, Undo, Bytecodes, UFNs) with live demos of running the software and documentation
-	Bibliographies (as “living”) for Notecards, Rooms, Loops, Common Loops, CLOS
And 5 other subtasks or spin-off projects to make “Interlisp” interesting and relevant to historians

The big picture is “escape velocity” (not just a sub-orbital jaunt). 

The “startup bylaws” is what was called a “forcing function”. I wanted to talk
about “what we’re doing and why”.  That’s the main point. The wrinkle
that ‘the result must read well in bylaws” is just there to help focus
the conversation. There are other routes we could take – a project in
cooperation with some other (non-profit) institutions. 
But without a clear description of what we're doing and why
nothing will happen.

**Comment by masinter** [comment]
In worrking Issue #7 Sort out LispUsers, consider

> The Software Preservation Network (SPN) Metadata Working Group is pleased to release the Software Metadata Recommended Format (SMRF) Guide. SMRF aims to provide an adaptable framework to describe a range of software types to support discovery, access, and reuse in libraries, museums, archives, and repositories.

You can find a link to the SMRF Guide on the SPN website: https://www.softwarepreservationnetwork.org/smrf-guide/

@ekaltman what do you think?

**Reply by ekaltman** [reply]
I'm not totally following here. Are you thinking of using the SMRF guide to describe this project correctly in textual references? I haven't read over the guide at this point. You also mention "living" bibliographies to link to various other projects, which is sounding like a semantic web use case.

**Comment by masinter** [comment]
To explain my point of view, I'll point to this 2006 paper
   https://larrymasinter.net/0603-archiving.pdf 
section 2.6 describes the issues with using emulation as a way of extending longevity of "born digital" material.

A part of the Interlisp trove of source code there are components and libraries and "hacks" (graphical material) written in Interlisp and contributed by users. We called this collection "Lispusers". We  are in the process of sorting out lispusers between things that run and can be demoed and things that are hopeless. I was looking for some guidelines for how to catalog software libraries like this, such as user contributions to DECUS.
